tests/controllers/test_crop_controller.py

```diff
@@ -37,14 +40,14 @@ def test_setup_crops_material_name(self):
         input_data = YamlReader().read_file(self.test_file)
 
         crop_controller = CropController(input_data, self.collection)
-        material, segmentation_id = crop_controller.assign_crop_type("red")
+        material, segmentation_id = crop_controller.assign_crop_type("stage1")
```

Comment:
It would be best to assign materials to all of the stages and check them all in a loop.